packages/eio/eio.0.4/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "Effect-based direct-style IO API for OCaml"
+description: "An effect-based IO API for multicore OCaml with fibers."
+maintainer: ["anil@recoil.org"]
+authors: ["Anil Madhavapeddy" "Thomas Leonard"]
+license: "ISC"
+homepage: "https://github.com/ocaml-multicore/eio"
+doc: "https://ocaml-multicore.github.io/eio/"
+bug-reports: "https://github.com/ocaml-multicore/eio/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "5.0.0~alpha0"}
```

Comment:
There is no such versions of the compiler. Every alpha version is behind the ocaml.5.0.0 virtual package
```suggestion
  "ocaml" {>= "5.0.0"}
```